Makefile

```diff
@@ -8,7 +8,7 @@ SDSL_DIR=../sdsl-lite
 #VERIFY_FLAGS=-DVERIFY_CONSTRUCTION
 
 # Multithreading with OpenMP and libstdc++ Parallel Mode.
-PARALLEL_FLAGS=-fopenmp -pthread -D_GLIBCXX_PARALLEL
+PARALLEL_FLAGS=-fopenmp -pthread
```

Comment:
There's probably a better way of doing this so that the flag is only enabled when using gcc.